internal/adapter/imagescanning/adapter_test.go

```diff
@@ -2,30 +2,103 @@
  * Copyright 2021 VMware, Inc.
  * SPDX-License-Identifier: Apache-2.0
  */
-package imagescanning
+ package imagescanning
```

Comment:
There's a leading space here. This can cause issues with some Golang tooling. Can you remove it?

Reply:
removed leading space